packages/elements/src/components/ino-popover/ino-popover.tsx

```diff
@@ -81,26 +94,41 @@ export class Popover implements ComponentInterface {
     this.create();
   }
 
-  // Private methods
+  /**
+   * Emits when an element which is not part of the popover is clicked.
+   * Should be used if you control the state of the popover.
+   */
+  @Event() clickOutside: EventEmitter<void>;
 
   private create() {
-    this.tooltipInstance?.destroy();
+    this.tippyInstance?.destroy();
 
     const target = this.inoFor
       ? document.getElementById(this.inoFor)
       : this.el.parentElement;
 
-    const options = {
+    if (!target && this.inoFor) {
+      console.warn(
+        `The element with the id '${this.inoFor}' could not be found.`
+      );
+    }
+
+    const isControlled = this.inoShow !== undefined;
+
+    const options: Partial<Props> = {
       allowHTML: true,
       appendTo: this.el.parentElement,
```

Comment:
Does this still make sense? I changed it to `'parent'` in order to place it as parent of the target, not the popover element.